### PR TITLE
Showcase October 2021

### PR DESCRIPTION
**Pull request by deccer**

### CODE REVIEW

File: showcase/2021/october/index.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	<script>markdeepOptions = { mode: 'script', tocStyle: 'none' };</script>
+	<script src="/scripts/markdeep.min.js"></script>
+
+	<title>October 2021 Showcase</title>
+
+	<!-- Open Graph Metadata -->
+	<meta property="og:title" content="October 2021 Showcase" />
+	<meta property="og:type" content="website" />
+	<meta property="og:url" content="https://graphics-programming.org/showcase/2021/october/" />
+	<meta property="og:image" content="https://imgur.com/JvHWTv6.png" />
```
Comment (nickclark2016) [inline showcase/2021/october/index.html]: https://imgur.com/EAz9VzK.png

Use this image here as well.  It should reflect the preview in the showcase (this is what's used for the discord preview)

File: showcase/2021/october/index.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+<body>
+	<code style="display:none" id="content">
+		# October 2021 Community Creation Showcase
+
+		<nav class="mx-auto text-center">
+			[Home](/) &bull; [Resources](/resources) &bull; [Showcase](/showcase)
+		</nav>
+
+		<div>
+			<video controls width="100%">
```
Comment (nickclark2016) [inline showcase/2021/october/index.html]: Use the markup for videos from Imgur.  You can see previous months.

**Review by nickclark2016 — CHANGES_REQUESTED** [review]

File: showcase/2021/october/index.html (hunk 1 above)
Comment (deccer) [reply showcase/2021/october/index.html]: Fixed.

**Review by nickclark2016 — APPROVED** [review]

File: showcase/2021/october/index.html (hunk 2 above)
Comment (deccer) [reply showcase/2021/october/index.html]: Fixed.

**Pull request merged**